src/main/java/io/reactivex/internal/operators/observable/ObservableBufferExactBoundary.java

```diff
@@ -17,6 +17,7 @@
 import java.util.concurrent.Callable;
 
 import io.reactivex.*;
+import io.reactivex.ObservableSource;
```

Comment:
Yup IntelliJ is not smart here and automatically inserts the import. Could also be because I've configured it to avoid * imports ...

Once I'm done with all of the renaming I could fix all of the imports if this is wanted.